### ISSUE TITLE
Triage team

### ISSUE THREAD

**Issue by wdhdev** [publicsuffix/list#2231]
I was thinking, what if we had a sort of triage team who labels PRs/issues? The main benefits would be it would reduce what maintainers have to do to very little, mainly only merging and possibly commenting on some issues.

If possible, could @groundcat and myself could get triage permissions, so when we review PRs/comment on issues, we can label them accordingly?

**Comment by simon-friedberger** [comment]
I support this. You're doing good work and if you could add labels the ticket states would be more obvious.

**Comment by weppos** [comment]
To confirm, you want @wdhdev and @groundcat to be added to the existing Triage team?

**Comment by simon-friedberger** [comment]
@weppos I think so but I cannot see team permissions so you will have to decide.

(You can, of course, make me an admin, then I can take care of it.)

**Comment by wdhdev** [comment]
@weppos @simon-friedberger FWIW, if it is a team on the organisation, I believe we would need to be added to the organisation so we can be added to the team, as I don't believe external members can be added to organisational teams.

Or you can manually set the triage permission per individual user in the repo settings.

**Comment by wdhdev** [comment]
Hey all, any updates on this?

**Comment by dnsguru** [comment]
ok so #2250 and #2248 give me some pause 

These appeared to have changes being submitted to IANA delegated ccTLDs _as if_ they were changes from the ccTLD, but instead were added from non-trusted third party sources either site:.foo or wikipedia.

ccTLDs need to specifically ONLY be updated by the ccTLD administrator or have their info updated to synchronize with a clearly verifiable source that follows the URL visiting path from IANA.org to the respective registry listed in the IANA TLD DB, and then cites the page on that registry which lists that namespace change.

Before we open up to more people with essentially merge rights, I want to push us to preserve the integrity of the changes and what happens in the ICANN section at the top section.

**Comment by dnsguru** [comment]
> I support this. You're doing good work and if you could add labels the ticket states would be more obvious.
> 
> (The team already exists, afaik)

I agree good work is happening but there was some room to improve before opening this up.

**Comment by wdhdev** [comment]
> ok so #2250 and #2248 give me some pause
> 
> These appeared to have changes being submitted to IANA delegated ccTLDs _as if_ they were changes from the ccTLD, but instead were added from non-trusted third party sources either site:.foo or wikipedia.
> 
> ccTLDs need to specifically ONLY be updated by the ccTLD administrator or have their info updated to synchronize with a clearly verifiable source that follows the URL visiting path from IANA.org to the respective registry listed in the IANA TLD DB, and then cites the page on that registry which lists that namespace change.

That is understandable, I apologise on my part.

I will attempt to contact the ccTLD operators to see if they can confirm if any updates to their sections are required.

In future, I'll ensure to get these updates from trusted sources (the operators themselves or the registry website.)

> I agree good work is happening but there was some room to improve before opening this up.

Do you have any suggestions on how I/we can improve?

**Comment by groundcat** [comment]
> Before we open up to more people with essentially merge rights, I want to push us to preserve the integrity of the changes and what happens in the ICANN section at the top section.

@dnsguru I agree with you that any changes made to the ICANN section should either use authoritative sources (starting from IANA database) or be confirmed by the registry, not by any third-party sources.

Regarding triaging, according to [GitHub documentation on Repository roles for an organization](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization), I think the triage permission does not allow merging a PR and does not include any write permission to the PSL text file. Therefore, the decision to merge or not would still remain with maintainers who have write permission, as a safeguard for the list. The triage permission is restrictive but sufficient for contributors adding tags to issues/PRs to improve efficiency without compromising the list’s integrity due to potential oversights.

**Comment by mozfreddyb** [comment]
@dnsguru The ccTLD case you are mentioning is supported and widely understood as important process. I am confident that this won't happen again. The question here is about triage access. All we want is that our new volunteers can help set lables and pre-check issues so the burden on the people that review & merge is further reduced. Can you please reconsider?

**Comment by groundcat** [comment]
hey @wdhdev, just so you know, I have been appending issue #1119 to each of the recent debris cleanup inquiry comments. This is to (hopefully) ensure I do not lose track of which ones I have already asked about, given that our tagging permissions are still pending under #2231. This method is a temporary way for tracking purposes. Happy New Year.

**Comment by wdhdev** [comment]
I noticed, good idea, I'll start doing that too. Happy (early) New Year!
